components/Starknet/modules/ROOT/pages/index.adoc

```diff
@@ -1,7 +1,189 @@
 [id="overview"]
 = Overview
 
-Starknet is a permissionless, decentralized validity rollup, also known as a zero-knowledge rollup (ZK rollup). It operates as a Layer 2 (L2) network over Ethereum, enabling any dApp to achieve unlimited scale for its computation - without
-compromising Ethereum’s composability and security - thanks to Starknet’s reliance on the safest and most scalable cryptographic proof system: STARK.
+Starknet is a permissionless, decentralized zero-knowledge rollup (ZK rollup) for Ethereum. It operates as a Layer 2 (L2), enabling any dApp to achieve unlimited scale for its computation without compromising on Ethereum’s composability
+and security.
 
-Starknet contracts and the Starknet OS are written in Cairo – supporting the deployment and scaling of any use case, whatever the business logic.
+Starknet achieves low cost transactions and high performance by using its own cryptographic proof system called STARK. Starknet contracts and the Starknet OS are written in link:https://github.com/starkware-libs/cairo[Cairo], a custom-built and
+specialized programming language.
+
+== Explore Starknet
+
+[pass]
+++++
+<div class="home-cta-container">
+  <a href="https://docs.starknet.io/documentation/quick_start/environment_setup/" class="home-cta home-cta-first" id="cta1">
+    <div class="image-container">
+      <img src="_images/developers.svg" style="filter: none; border-radius: 0px;" class="cta-image">
+    </div>
+    <h2>Quick start</h2>
+    <p class="chakra-card__body css-jintet">Set up your environment and get started with Starknet.</p>
+    <p class="chakra-card__body css-jintet">Explore ></p>
+  </a>
+
+  <a href="https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/header/" class="home-cta" id="cta2">
+    <div class="image-container">
+      <img src="_images/how_SN_works.svg" style="filter: none; border-radius: 0px;" class="cta-image">
+    </div>
+    <h2>Architecture</h2>
+    <p class="chakra-card__body css-jintet">Learn about Starknet's architecture including contracts and accounts.</p>
+    <p class="chakra-card__body css-jintet">Explore ></p>
+  </a>
+
+  <a href="https://docs.starknet.io/documentation/starknet_versions/version_notes/" class="home-cta" id="cta3">
+    <div class="image-container">
+      <img src="_images/roadmap.svg" style="filter: none; border-radius: 0px;" class="cta-image">
+    </div>
+    <h2>Releases</h2>
+    <p class="chakra-card__body css-jintet">Learn more about the current version of Starknet.</p>
+    <p class="chakra-card__body css-jintet">Explore ></p>
+  </a>
+</div>
+++++
+
+== Developer tools and resources
+
+[pass]
```

Comment:
I think you can achieve the same formatting here using AsciiDoc tables. https://docs.asciidoctor.org/asciidoc/latest/tables/borders/

Did you already try this before resorting to HTML?

Reply:
Yes, due to other CSS in the theme it's not straight forward to style this way using the native asciidoc tables